addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -90,7 +90,14 @@ var LivechatButton = Widget.extend({
                     self._history = history;
                 });
         }
-        return ready.then(this._loadQWebTemplate.bind(this));
+        session.session_bind();
```

Comment:
I believe this will call `bootstrap_translations` which will load the translations for the `web` module while they are not required.

Reply:
Okay, actually it's enough if session.user_context is initialized, so we can just do that